src/main/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtils.java

```diff
@@ -74,6 +75,7 @@ public static boolean isCheckstyleModule(Class<?> clazz) {
                     || isFileSetModule(clazz)
                     || isFilterModule(clazz)
                     || isFileFilterModule(clazz)
+                    || isTreeWalkerFilterModule(clazz)
```

Comment:
https://github.com/checkstyle/checkstyle/pull/4854#discussion_r130225527
> Also please update javadoc to include the new module.

This point was not done.